docs/how-to-guides/own_machine/clone-renku-project.rst

```diff
@@ -0,0 +1,116 @@
+.. _clone_renku_project:
+
+Clone a Renku project
+=====================
+
+In order to save your work back to RenkuLab from your own machine, you'll need to authenticate with RenkuLab.
+You can do this easily by the ``renku login`` command.
+
+However, if you find yourself needing to authenticate via an SSH Key, you can find those instructions below as well.
+
+.. tabbed:: Authenticate via renku login
+
+    To authenticate with Renku on a local machine where you have access to a browser, you can use :meth:`renku login <renku.ui.cli.login>`.
+
+    Provide the url of the RenkuLab instance where your project is located in the following command:
+
+    .. code-block:: console
+
+        $ renku login <url of your renkulab instance>
+
+    For example, if your project is on https://renkulab.io , enter:
+
+    .. code-block:: console
+
+        $ renku login renkulab.io
+
+    This will open RenkuLab in a browser window, where you can enter your credentials.
+    Then, the local Renku CLI receives and stores a secure token that will be used for future authentications.
```

Comment:
```suggestion
    This will open RenkuLab in a browser window, if possible, where you can enter your credentials.
    If it is not possible to open a browser (e.g. because you are logged in to a remote server) you can copy/paste the URL it displays to complete the login. 
    Then, the local Renku CLI receives and stores a secure token that will be used for future authentications.
```